omise/classes/charge.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
@nimid if it's not break any PrestaShop rules, is it possible to have file name to `Charge.php` to reflect its class name?